src/hearings/containers/linked-hearings/linked-hearings-with-case/linked-hearings-with-case.component.ts

```diff
@@ -0,0 +1,97 @@
+import { HearingListMainModel } from '../../../models/hearingListMain.model';
+import { EXUIDisplayStatusEnum } from '../../../models/hearings.enum';
+import { HearingDetailModel, ServiceLinkedCasesModel } from '../../../models/linkHearings.model';
+import { HearingsService } from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+
+@Component({
+  selector: 'exui-linked-hearings-with-case',
+  templateUrl: './linked-hearings-with-case.component.html',
+  styleUrls: ['./linked-hearings-with-case.component.scss']
+})
+export class LinkedHearingsWithCaseComponent implements OnInit, OnDestroy {
+  public linkedCases: ServiceLinkedCasesModel[];
+  public sub: Subscription;
+  public linkHearingForm: FormGroup;
+  public caseTitle: string;
+
+  constructor(private readonly hearingStore: Store<fromHearingStore.State>,
+              private readonly hearingsService: HearingsService,
+              private readonly route: ActivatedRoute,
+              private readonly fb: FormBuilder) {
+  }
+
+  public addHearingFormGroup(caseRef: string): FormGroup {
+    return this.fb.group({
+      caseReference: caseRef,
+      hearingReference: new FormControl(),
+    });
+  }
+
+  public ngOnInit(): void {
+    this.linkHearingForm = this.fb.group({
+      hearings: this.fb.array([])
+    });
+    this.getAllCaseInformation();
+  }
+
+  public getAllCaseInformation() {
+    const receivedCases: ServiceLinkedCasesModel[] = this.route.snapshot.data.linkedCase && this.route.snapshot.data.linkedCase.serviceLinkedCases || [];
+    const linkedCaseIds: string[] = receivedCases.map((caseDetails: ServiceLinkedCasesModel) => caseDetails.caseReference);
+    const hearingServices = [];
+    linkedCaseIds.forEach(id => {
+      hearingServices.push(this.hearingsService.getAllHearings(id));
+    });
+    this.sub = forkJoin(hearingServices).subscribe((hearingsList: HearingListMainModel[]) => {
+      this.linkedCases = receivedCases.map((caseInfo: ServiceLinkedCasesModel, pos: number) => {
+        (this.linkHearingForm.get('hearings') as FormArray).push(this.addHearingFormGroup(caseInfo.caseReference));
+        const hearings = [] as HearingDetailModel[];
+        hearingsList[pos].caseHearings.forEach((hearing) => {
+          if (hearing.exuiDisplayStatus.toLowerCase() === EXUIDisplayStatusEnum.AWAITING_LISTING.toLowerCase() || hearing.exuiDisplayStatus.toLowerCase() === EXUIDisplayStatusEnum.UPDATE_REQUESTED.toLowerCase()) {
+            const hearingInfo: HearingDetailModel = {
+              hearingId: hearing.hearingID,
+              hearingStage: hearing.hearingType,
+              isSelected: false,
+              hearingStatus: hearing.exuiDisplayStatus,
+              hearingIsLinkedFlag: hearing.hearingIsLinkedFlag
+            };
+            hearings.push(hearingInfo);
+          }
+        });
+        return {
+          hearings,
+          ...caseInfo
+        };
+      });
+    });
+  }
```

Comment:
We may need to move this to Node layer later and retreive the data directly in ONE node API. But for time being agreed we will leave it, and refactor later if needed.